grails-app/domain/au/org/ala/ecodata/Project.groovy

```diff
@@ -120,6 +120,11 @@ class Project {
 
     List<OutputTarget> outputTargets
 
+    //New fields to be used bt new search page (initiator=ala response of /ws/project/search)
```

Comment:
Please remove all these fields from domain object. Add them as fields to projectMap in ElasticSearchService.